tasklist/client/e2e/tests/task-details.spec.ts

```diff
@@ -18,6 +18,12 @@
 import {expect} from '@playwright/test';
 import {test} from '../test-fixtures';
 import {createInstances, deploy} from '../zeebeClient';
+import {
+  fillDate,
+  selectDropdownValue,
+  enterTime,
+  selectTaglistValues,
+} from '../pages/UtilitiesPage';
```

Comment:
Let's inject the page object model like we're doing with other page object classes instead of importing it like this